Aweseome work!

PumaBlack opened the pull request:
Dear darkevilmac:

At first, THANK YOU for updating and making the AS Mod better than before.

I freaked out, seeing that logistic pipes are working now on my ship. I have to be patient for a minute, but then: whohaa! all connections go green, they work fine.

Than the rotation, almost every block from other mods remember there alignment after the unmonut of my ship. The drawbridge from tinkers construct fails, but hey - this will maybe get fixed ;)

Even Thermal Expansion Pipes work almost(!) fine, a few connection bugs between chests and pipes i'd figured out. Awesome!

Project Red is not possible at all (Gates and wires). But hey - again: THANK YOU for taken time to push this great mod forward.

THUMBS UP!

darkevilmac committed:
Some work on adding the ship anchor for aligning to certain points in the world. Very basic, not even close to working.

darkevilmac committed:
Update README.md

darkevilmac committed:
Update Gitignore, Format some code (Need full reformat), Work on BlockAnchorPoint. Currently crashes game when ship attempts to lock onto an anchor.

darkevilmac committed:
Merge branch 'master' of https://github.com/darkevilmac/archimedes-ships-plus

darkevilmac committed:
Anchor Point fully implemented. Major Reformat, method of finding anchor points could be much better but it's functional. This build is not to be used ships WILL despawn when a world is reloaded.

darkevilmac committed:
Changed anchorpoint algorithm, now using AnchorPointData in the ship entity instead of the tileentity, having issues with AnchorPoint NBT most likely something silly.

darkevilmac committed:
Anchor is working wonderfully, they go to the nearest anchor to them. AnchorPoint NBT issues fixed. Slight changes to the block code.

darkevilmac committed:
Fixed the last of the bugs with the anchorpoint. Added AssembleBlockEvent.

darkevilmac committed:
Change mod name references to Archimede's Ships Plus instead of Archimede's Ships. Changed author in build.properties.

darkevilmac committed:
Change Recipe for AnchorPoint, change license.

darkevilmac committed:
Change License to QPL 1.0

darkevilmac committed:
AGPLV3

darkevilmac committed:
Update README.md

darkevilmac committed:
Add movingworld submodule.

darkevilmac committed:
Add way more information for devs to the readme.

darkevilmac committed:
Not functional but I need to make the repo look like there's actual things going on because there are.

darkevilmac committed:
Merge branch 'master' of https://github.com/darkevilmac/archimedes-ships-plus

darkevilmac committed:
Update forge.

darkevilmac committed:
Remove /docs/ remove build scripts. A little code.

darkevilmac committed:
Add SetupWorkspace.bat

darkevilmac committed:
i am gud at batch

darkevilmac committed:
ckathode -> darkevilmac, Work on networking leak fix, TileMovingWorldMarker instead of Helm, etc.

darkevilmac committed:
We can launch now, so that's pretty cool.

darkevilmac committed:
Some work on client syncing.

darkevilmac committed:
Attribution-NonCommercial-ShareAlike 4.0 International

I dislike GNU licenses, so I'm back to this.

darkevilmac committed:
Assembly works (sort of) fixed some network stuff.

darkevilmac committed:
Update MovingWorld submodule for networking fixes.

darkevilmac committed:
Merge branch 'master' of https://github.com/darkevilmac/archimedes-ships-plus

darkevilmac committed:
Add Side parameter to messages, ships now assemble they don't turn to entities yet but they assemble as expected.

darkevilmac committed:
Ship mounting working? Ship mounting working.

darkevilmac committed:
Ships can now decompile, and compile.

darkevilmac committed:
Ships move again, not up and down though.

darkevilmac committed:
I believe I can fly.

darkevilmac committed:
Ships render and spawn properly on all clients connected to a server, next goal is adding back seats.

darkevilmac committed:
Update MovingWorld Submodule

darkevilmac committed:
Untested, seats seem to work though.

darkevilmac committed:
Update MovingWorld Submodule

darkevilmac committed:
If you disconnect when you're on a ship seat it's weird, if you're not on the seat. Iz gud. (Needs to be fixed.)

DeRigeurEx committed:
Create cs_CZ.lang

Create "cs_CZ.lang" from "en_US.lang"

DeRigeurEx committed:
Update cs_CZ.lang

Translated all.
row 20 FIX - "light_blue" to "lightBlue" 
Tested at [1.7.10-10.13.0.1198]ArchimedesShipsPlus-universal-1.7.10.B-Plus.jar

darkevilmac committed:
Merge pull request #12 from DeRigeurEx/patch-1

Patch 1 -row 20 FIX- for *.lang

darkevilmac committed:
Update en_US.lang

light_blue -> lightBlue

darkevilmac committed:
Update fr_FR.lang

light_blue -> lightBlue

darkevilmac committed:
Update nl_NL.lang

light_blue -> lightBlue

darkevilmac committed:
Update pt_BR.lang

light_blue -> lightBlue

darkevilmac committed:
Update ru_RU.lang

light_blue -> lightBlue

darkevilmac committed:
Update zh_CN.lang

light_blue -> lightBlue

darkevilmac committed:
Seats fully functional, and when I say fully I mean FULLY.

darkevilmac committed:
Merge branch 'master' of https://github.com/darkevilmac/archimedes-ships-plus

darkevilmac committed:
Remove SetParentMessages, format and organize imports.

darkevilmac committed:
Some config stuff removed. Update MovingWorldSubModule.

darkevilmac committed:
I think I finally fixed seats. There's one bug I'm seeing but it's low priority, when you're in a seat and someone disconnects you stay in the seat floating in the air. Just dismount and you're good, so it's low priority.

darkevilmac committed:
Add indev calls, change DisconnectHandler to ConnectionHandler. Seats work well enough, they're by no means perfect but they work okay.

darkevilmac committed:
Engines are back, and they don't desync \o/

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
Add in mandatory engines config.

darkevilmac committed:
Ships now fall if they have no fuel and engines are mandatory.

darkevilmac committed:
Update Forge, Update MovingWorld, Fix #13, MROT files pre installed. UNTESTED IN COMPILED ENVIRONMENT!

darkevilmac committed:
Works in compiled environment.

darkevilmac committed:
Fix #11

darkevilmac committed:
Fix #16

darkevilmac committed:
Format, Optimize Imports, Anchor Point functioning again, changed anchor messages to use lang.

darkevilmac committed:
bump version to alpha-0.1, update movingworld

darkevilmac committed:
Update logo, update mcmod.info, update MovingWorld, bump to beta.

darkevilmac committed:
Bump forge version to 1.8-11.14.1.1404, Update MovingWorld

darkevilmac committed:
Clean EntityShip of errors, clean EntityParachute of errors, Update MovingWorld submodule.

darkevilmac committed:
Remove HelmRenderer, add MovingWorld Submodule. Lots more, not working yet.

darkevilmac committed:
Bump forge, update MovingWorld submodule, change some render code, add some models for the gauge and engine.

darkevilmac committed:
fix engine model

darkevilmac committed:
Fix some broken imports, add experimental seat model.

darkevilmac committed:
update gitignore to include gradle gui for 64bit windows

darkevilmac committed:
Auto remap blocks, adds greater compat with ArchimedesShipsLegacy.

darkevilmac committed:
1.7.11

darkevilmac committed:
Update MovingWorld Submodule, Fix remaining import issues, game compiles, model files needed for other blocks.

darkevilmac committed:
Move assets, seat model working, add missing overrides.

darkevilmac committed:
Fix placement of gauges state issues, remove ability to place gauge on fences.

darkevilmac committed:
Fix odd rendering with gauge seen here: https://imgrush.com/7W1jLFOM-0j5

darkevilmac committed:
Remove commented old code.

darkevilmac committed:
remove gauge_extended model, remove unneeded RenderParachute line, auto register simple item renderers.

darkevilmac committed:
Make balloons recolourable.

darkevilmac committed:
Most models render now.

darkevilmac committed:
Update MovingWorld.

darkevilmac committed:
Fix crash on clients, bump version to 1.7.12

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
Fix texture location references, format, optimize imports.

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
frontDirection is an EnumFacing, adjusted accordingly.

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
Update MovingWorld, getHorizontalIndex.

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
Update forge version and MCP mappings, adjust accordingly.

darkevilmac committed:
Make BlockRecolourable make more sense (im from canada eh COLOUR with a U)

darkevilmac committed:
Update MovingWorld, bump versions properly.

darkevilmac committed:
add .travis

TehStoneMan commented:
I don't think this should be considered as a pull request, as it appears to be replacing the entire original Archimedes' Ships with Darkevilmac's Plus version. Also, this doesn't seem to come from the code author. Maybe you just meant to leave a comment?

darkevilmac committed:
Misc fixes

darkevilmac committed:
Update MovingWorld, add mixins.

darkevilmac committed:
Add a quick fix to a nullpointer, Update MovingWorld submodule

darkevilmac committed:
Fix Gauge rendering in inventory as well as in the world.

darkevilmac committed:
Reformat, add final missing models, remove old testing 'balloon' folder.

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
Fix incorrect Vec3 class, use .getBlockMetaData because we don't have an accesstransformer.

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
Reformat some things, update MovingWorld.

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
Fix #28

darkevilmac committed:
Update MovingWorld, adjust build.gradle to match MovingWorld

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
Merge branch 'master' into 1.8

darkevilmac committed:
Correct coremod package location in VM args.

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
See https://github.com/darkevilmac/MovingWorld/commit/6f8feb06c08b0636c3cc40f1a8b2d22ca3ed83c4

darkevilmac committed:
Major refactor and reorganizing

darkevilmac committed:
Blocks and Items moved to common.object, registration of blocks and items moved from the Mod class, reformat and optimize imports.

darkevilmac committed:
Adjust block instance locations to match refactor.

darkevilmac committed:
The console did nothing wrong.

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
Camel case.

darkevilmac committed:
NoClassDef for some reason?

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
Misleading config option, MovingWorld updated.

darkevilmac committed:
Add StickyBuffer stub, missing blockstate files. Add a resource for the wiki.

darkevilmac committed:
Add logo to wiki.

darkevilmac committed:
Add more wiki images.

darkevilmac committed:
Add StickyBuffer and Anchor wiki image.

darkevilmac committed:
Fix whitespace in anchor image.

darkevilmac committed:
Add balloon, change incorrect helm naming.

darkevilmac committed:
Update MovingWorld for major bug fixes.

darkevilmac committed:
Alpha-1, Update MovingWorld.

darkevilmac committed:
Update MovingWorld, add balloon alternatives to config.

darkevilmac committed:
Initialize balloon alternatives in configs, fix balloon alternatives causing crash.

darkevilmac committed:
Add ability to change configs in game.

darkevilmac committed:
Fully implement sticky buffer, add whitelist blocks.

Update MovingWorld

darkevilmac committed:
Bump to Alpha-2, add MovingWorld_at so it compiles consistently, camelCase for anchor point model

darkevilmac committed:
Ignore Compilation.

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
Update MovingWorld, fix incorrect casting assumptions, auto change mod version to match build.gradle

darkevilmac committed:
Close #20, anchors align properly

darkevilmac committed:
Add submersible button to ship gui, implement FloodFill, Update MovingWorld.

darkevilmac committed:
Basic Submersible implementation.

darkevilmac committed:
Update MovingWorld, some more work on submersible ships.

darkevilmac committed:
Adjust to MovingWorld Refactor.

darkevilmac committed:
Fix #33

darkevilmac committed:
Add model for helm with rotation based on ship movement.

darkevilmac committed:
Update MovingWorld, remove debug print.

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
Mark version to Alpha-3.0, Update MovingWorld

darkevilmac committed:
Fix #35

darkevilmac committed:
Switch back to having our own mass variable.

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
Fix submerge bounciness with ships. Close #36

darkevilmac committed:
Fix flying submersibles, force submersibles to sink.

Bump to A3.1

darkevilmac committed:
Update MovingWorld, cache helm wheel model so we don't bake it every frame.

darkevilmac committed:
Implement ITileEngineModifier.

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
Implement IBlockCustomMass, and IBlockBalloon.

darkevilmac committed:
Rotate crates, add creativetab, fix crate model, bump version

darkevilmac committed:
Close #38, Close #41

darkevilmac committed:
Update MovingWorld, bump to stable 1.8 version number.

darkevilmac committed:
Add standalone submersible icon for reference. Add reference assets to gitignore.

darkevilmac committed:
Update README.md

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
Correct typo

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
Close #51

darkevilmac committed:
Add Contributing file, Close #39 thanks @JCMais

darkevilmac commented:
I think he did, I ask that this is closed unless you want a complete re-license :P

Feels pretty cool looking at all those commits I've made though. And that's not even including the MovingWorld changes.

Not to sound rude but I'm not really sure why people are still making PRs to the original mod, I'm starving for developers on my end ;-;

darkevilmac committed:
Basic secured bed implementation. (Not Tested, likely non functional)

Update MovingWorld

darkevilmac committed:
Update Forge, Minor works to the SecuredBed (It works but not fully yet).

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
Update LICENSE

darkevilmac committed:
Update LICENSE

darkevilmac committed:
Update LICENSE

darkevilmac committed:
Remove some old unneeded files, update to ForgeGradle 2, ignore build folder.

darkevilmac committed:
Merge branch 'master' of http://github.com/darkevilmac/archimedes-ships-plus

darkevilmac committed:
Minor Changes.

Update MovingWorld

darkevilmac committed:
Update CONTRIBUTING.MD

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
Merge branch 'master' of http://github.com/darkevilmac/archimedes-ships-plus

darkevilmac committed:
Experimental network config serialization.

Update MovingWorld

darkevilmac committed:
Remove useNewAlgorithm because it didn't do anything. Modify all direct config references to a getter for a synced config.

arcriley commented:
> Not to sound rude but I'm not really sure why people are still making PRs to the original mod, I'm starving for developers on my end ;-;

@darkevilmac Many people continue to use the original Archimedes Ships since its available under the MIT license and thus does not require permission to redistribute or fork.

If you want more people to contribute to your fork then release it under a FOSS license. Your choice in licensing is driving away both users and potential contributors.

darkevilmac committed:
Implement secured bed properly, Update MovingWorld, fix Network Config bugs when playing in singleplayer as well as serialization issues in multiplayer.

darkevilmac committed:
Update to 1.8.9

darkevilmac committed:
Update MovingWorld.

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
Update LICENSE

darkevilmac committed:
Update LICENSE

darkevilmac committed:
Update MovingWorld, adjust mcmod.info, move entity render registration to preinit.

darkevilmac committed:
Merge branch 'master' of http://github.com/darkevilmac/archimedes-ships-plus

darkevilmac committed:
Cleanup Code, Update MovingWorld

darkevilmac committed:
Fix the rest of the major bugs on 1.8.9

darkevilmac committed:
Update MovingWorld

darkevilmac committed:
Update MovingWorld, Update Gradle, Update Travis

darkevilmac committed:
Actually update travis.

darkevilmac committed:
Update LICENSE

darkevilmac committed:
Add German language file placeholder.

darkevilmac committed:
Merge branch 'master' of http://github.com/darkevilmac/archimedes-ships-plus

SetsunaLP committed:
Update de_DE.lang

I've translated the words into german, that you can make a german version of your mod.

Could be, that something isn't translated good enough. I'll see if I can translate something better after you introduced it.

darkevilmac committed:
Merge pull request #72 from SetsunaLP/patch-1

Update de_DE.lang

darkevilmac committed:
Attempt to fix some nonsense.

darkevilmac committed:
Update MovingWorld!

darkevilmac committed:
Add submersible non functioning icon, port some config options from 1.7.10, placeholder for name badge stuff.

darkevilmac committed:
Update MovingWorld, this should fix #74.

darkevilmac committed:
Close #69 #68 #74

darkevilmac committed:
Attempt to fix netwokring bug I can't reproduce, Update MovingWorld

darkevilmac committed:
Begin 1.9 update

darkevilmac committed:
Some changes here and there, slow progress but progress nonetheless. Current thoughts, collision helper methods are nice, mounting changes to accompany boats are not so nice.

darkevilmac committed:
Update MovingWorld, Update Mappings.

darkevilmac committed:
Update MovingWorld and build.gradle.

darkevilmac committed:
1.9 updates, we're getting there. 100 errors left.

darkevilmac committed:
And we compile!

darkevilmac committed:
Fix some models, add onMainThread checks for packets, fix packet handler, fix models, fix crates and let them be redstone powered.

darkevilmac committed:
Redstone input crates.

darkevilmac committed:
Update MovingWorld.

darkevilmac committed:
Update CONTRIBUTING.MD

darkevilmac committed:
Update CONTRIBUTING.MD

darkevilmac committed:
Delete README.txt
